Resources/Prototypes/_LostParadise/Loadouts/Jobs/justice.yml

```
@@ -0,0 +1,36 @@
+
+- type: loadout
+  id: LoadoutMagazineBoxAntiMateriel
+  category: JobsCommand
+  cost: 1
+  exclusive: true
+  requirements:
+    - !type:CharacterJobRequirement
+      jobs:
+        - ChiefJustice
+  items:
+    - MagazineBoxAntiMateriel
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)